### ISSUE TITLE
Missing informations for building correctly

### ISSUE THREAD

**Issue by brugnara** [ubinity/webpcsc-firebreath#5]
First of all, thank you for this great plugin. :+1: 
I'm opening a issue for reporting a failure when building in windows (and a trivial issue on linux).

1) This file:
_$(FBPlugins)/projects/PCSCBridge/projectDef.cmake_

doesn't exists. The correct path is:

_$(FBPlugins)/projects/PCSCBridge/_ **_win/**_ _projectDef.cmake_

2) for building **PCSCBridge** project in visual studio, you also need to:
- add include path: C:\WinDDK\7600.16385.1\inc\atl71
- add lib path: C:\WinDDK\7600.16385.1\lib\ATL\i386
- new lib must be added too: ATLTHUNK.LIB (this solved my linker error, thanks to this forum: http://www.magpcss.org/ceforum/viewtopic.php?f=6&t=332 that helped me a lot :+1: 
  ps: I've just found those steps described in FireBreath compiling instruction http://www.firebreath.org/display/documentation/Building+with+Visual+Studio+Express

In linux, I wasn't able to view the smart card reader in test.html. I've solved installing **pcscd** (a simple apt-get install but isn't documented :smile: )

Daniele.

**Comment by ubinity** [comment]
Dear Daniele

Thanks for your interest in this plugin and reporting issue.

About windows, I will recheck the compilation and update the doc as soon as
possible (certainly before end of next week)

About linux, I already merged your push request about packages to install.

For your information and remainder, I work on a V0.3 version and as said in
the documentation the native TerminalAPI and ReaderAPI will be removed. I
invite you to base your developpement on the "scardjs" API (in
dist/js/scard.js). This js interface should be stable and will wrap
transparently (hopefully) any future native refactor

If you have any requests/needs for the V0.3 feel free to ask. And of course
all issue reports are welcome :)

Regards

Cédric

On 9 October 2013 16:35, Daniele notifications@github.com wrote:

> First of all, thank you for this great plugin. [image: :+1:]
> I'm opening a issue for reporting a failure when building in windows (and
> a trivial issue on linux).
> 
> 1) This file:
> _$(FBPlugins)/projects/PCSCBridge/projectDef.cmake_
> 
> doesn't exists. The correct path is:
> 
> _$(FBPlugins)/projects/PCSCBridge/_ _win/_ _projectDef.cmake_
> 
> 2) for building _PCSCBridge_ project in visual studio, you also need to:
> - add include path: C:\WinDDK\7600.16385.1\inc\atl71
> - add lib path: C:\WinDDK\7600.16385.1\lib\ATL\i386
> - new lib must be added too: ATLTHUNK.LIB (this solved my linker
>   error, thanks to this forum:
>   http://www.magpcss.org/ceforum/viewtopic.php?f=6&t=332 that helped me
>   a lot [image: :+1:]
> 
> In linux, I wasn't able to view the smart card reader in test.html. I've
> solved installing _pcscd_ (a simple apt-get install but isn't documented [image:
> :smile:] )
> 
> Daniele.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ubinity/webpcsc-firebreath/issues/5
> .

## 

Cédric Mesnil
R&D Team
Skype: cedric.mesnil  -  email: cedric.mesnil@ubinity.com

**Comment by brugnara** [comment]
That js file require plugin is installed, is that true?
ps: thanks for merging but the "pcscd" is another package i didn't reported in previous pull request. :ok_hand: 

Daniele.

**Comment by ubinity** [comment]
Yes that's true. It's just a user friendly wrapper to plugin. Also read the
doc dist/doc/PCSCBridge.html

pcscd added to the list

Cédric

On 10 October 2013 09:26, Daniele notifications@github.com wrote:

> That js file require plugin is installed, is that true?
> ps: thanks for merging but the "pcscd" is another package i didn't
> reported in previous pull request. [image: :ok_hand:]
> 
> Daniele.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/ubinity/webpcsc-firebreath/issues/5#issuecomment-26033859
> .

## 

Cédric Mesnil
R&D Team
Skype: cedric.mesnil  -  email: cedric.mesnil@ubinity.com
